ProjectLibertyLabs/social-app-template
# Feature: Add the ability to selectively start services

# Purpose
The goal of this PR is allow users to select a subset of services to launch.


## Solution

Updated `./start.sh` to ask whether each service should be started. Those answers are parsed and passed to the `docker compose` command. The values are also saved in the saved env file.

## Steps to Verify
1. Execute `./start.sh` 
2. When asked whether or not to start a service, answer anything other than "y", e.g. "n".
3. Verify that the excluded services are not started.

Note: frontend and backend have dependencies on all the other services, so if one of those are yes, all the rest must also be yes.
